collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
@cofyc In fact i am not sure the relation between client-go and `pkg/api/v1`. However, some functionalities are both available in these two packages.

In fact, there is a corresponding [`GetPersistentVolumeClaimClass`](https://github.com/kubernetes/client-go/blob/df46f7f13b3da19b90b8b4f0d18b8adc6fbf28dc/pkg/api/helper/helpers.go#L549) in client-go in `k8s.io/client-go/pkg/api/helper` package.

This will also needs to update Godeps.